routers/user.router.js

```diff
@@ -1,20 +1,25 @@
-const express = require('express');
-const userController = require('./user.controller');
-const authorization = require('../middlewares/auth');
+const express = require("express");
+const userController = require("./user.controller");
+const authorization = require("../middlewares/auth");
+const { userCurrent } = require("./user.controller");
```

Comment:
method should describe an action. We've talked about that

Reply:
getCurrentUser